protobuf_definitions/message_formats.proto

```diff
@@ -26,15 +26,20 @@ message BinlogRecord {
 // Used to indicate the desired motion in each direction.
 // Typically these values map to the left and right joystick for motion,
 // and the left and right trigger for the slow and boost modifiers.
+//
+// You can use the slow and boost modifiers to increase or decrease the speed
+// of the motion, from the default baseline.
```

Comment:
Maybe we could include some examples on how much the modifiers change the speed, eg. does boost=1 equal double the speed of just surge=1? And how does that work with slow, is slow=1 the same as surge = 0?

Reply:
A bit more complicated than that, they are not added directly on top of the input components but are used further down the stack in control system code. So it's hard to say exactly how much faster `1.0` is compared to `0.0`, other than saying "this is the maximum extra speed we allow the user to get", and is also an implementation detail as it will be different on different models (even if the protocol is the same).

Reply:
Agreed - What about for slow though?

Reply:
It's the same. I.e. `0` does not make it stop, and it's the same. We don't have a hard number (in newtons or meters per second) to how much slower it makes the drone.